analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/InsecureDeserializationTest.cs

```diff
@@ -28,20 +28,23 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class InsecureDeserializationTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder().AddAnalyzer(() => new InsecureDeserialization(AnalyzerConfiguration.AlwaysEnabled)).WithBasePath("Hotspots");
```

Comment:
The nice link in the test results that brings you to the line in the test file with the error does not work here. I think it is related to the "Hotspots" path. I did a note about that.